src/storage/errors.rs

```diff
@@ -364,6 +364,30 @@ pub fn extract_committed(err: &Error) -> Option<TimeStamp> {
     }
 }
 
+fn ensure_debug_info_for_key_error(err: &mut kvrpcpb::KeyError) -> &mut kvrpcpb::DebugInfo {
+    if err.debug_info.is_none() {
+        err.set_debug_info(kvrpcpb::DebugInfo::default());
+    }
+    err.debug_info.as_mut().unwrap()
+}
+
+fn add_debug_mvcc_for_key_error(
+    err: &mut kvrpcpb::KeyError,
+    key: &[u8],
+    mvcc_info: Option<types::MvccInfo>,
+) {
+    if let Some(mut mvcc) = mvcc_info {
+        let debug_info = ensure_debug_info_for_key_error(err);
+        // remove the values in default CF to reduce the size of the response.
+        mvcc.values.clear();
+        // set mvcc info to debug_info
+        let mut mvcc_debug_info = kvrpcpb::MvccDebugInfo::default();
+        mvcc_debug_info.set_key(key.to_owned());
+        mvcc_debug_info.set_mvcc(mvcc.into_proto());
+        debug_info.mvcc_info.push(mvcc_debug_info);
+    }
+}
+
```

Comment:
How about
```suggestion
fn add_debug_mvcc_for_key_error(
    err: mut kvrpcpb::KeyError,
    key: &[u8],
    mvcc_info: Option<types::MvccInfo>,
) -> kvrpcpb::KeyError{
    if let Some(mut mvcc) = mvcc_info {
        let debug_info = err.debug_info.get_or_insert_default();
        // remove the values in default CF to reduce the size of the response.
        mvcc.values.clear();
        // set mvcc info to debug_info
        let mut mvcc_debug_info = kvrpcpb::MvccDebugInfo::default();
        mvcc_debug_info.set_key(key.to_owned());
        mvcc_debug_info.set_mvcc(mvcc.into_proto());
        debug_info.mvcc_info.push(mvcc_debug_info);
    }
    err
}
```

Reply:
`SingularPtrField` seems do not support `get_or_insert_default`